backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/nuernberg/steps/SanitizeGeocode.kt

```diff
@@ -0,0 +1,56 @@
+package app.ehrenamtskarte.backend.stores.importer.nuernberg.steps
+
+import app.ehrenamtskarte.backend.stores.STREET_EXCLUDE_PATTERN
+import app.ehrenamtskarte.backend.stores.geocoding.FeatureFetcher
+import app.ehrenamtskarte.backend.stores.geocoding.isCloseToBoundingBox
+import app.ehrenamtskarte.backend.stores.geocoding.isInBoundingBox
+import app.ehrenamtskarte.backend.stores.importer.ImportConfig
+import app.ehrenamtskarte.backend.stores.importer.PipelineStep
+import app.ehrenamtskarte.backend.stores.importer.common.types.AcceptingStore
+import io.ktor.client.HttpClient
+import kotlinx.coroutines.runBlocking
+import org.geojson.Feature
+import org.geojson.Point
+
+/**
+ * Sanitize the postal code and the coordinates of the [AcceptingStore] using forward geocoding.
+ * If the coordinates are not inside the bounding box of the postal code, one of those is wrong.
+ * Then query by the address and use the coordinates OR postal code of the first match to sanitize the store data.
+ */
+class SanitizeGeocode(config: ImportConfig, httpClient: HttpClient) :
```

Comment:
Is this step different to Bayern's SanitizeGeocode step?
If not, maybe we can extract that out into the `common` steps (Although I guess you had some reason not to do that :) -- Maybe you can add a comment what the difference is)

Reply:
since we now have `long/lat` we can use them step as common